test/e2e/data/ironic-standalone-operator/ironic/overlays/e2e-release-27.0/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+namespace: baremetal-operator-system
+
+resources:
+  - ../../base
+
+components:
+  - ../../../components/tls
+  - ../../../components/basic-auth
+
+patches:
+  - target:
+      kind: Ironic
+      name: ironic
+    patch: |-
+      - op: replace
+        path: /spec/images/ironic
```

Comment:
Late realization, sorry: you should not patch this. Instead, patch in `version: "27.0"` etc